### ISSUE TITLE
how to change default app sharing option? 

### ISSUE THREAD

**Issue by starlite-jack** [K1rakishou/Kuroba-Experimental#635]
![Edited_20211116_195811~2](https://user-images.githubusercontent.com/90665508/141990967-0de84163-183a-407c-bf72-b71a4c841692.jpg)
as you can see in the picture above, I'm trying to change the default app for sharing media but I don't know how.

When I tried to share a picture by tapping "share media content" it always chooses my default app. I want to change that. any idea on how to do it?

**Comment by K1rakishou** [comment]
In your phone's settings. Go to the settings of the application that you selected as sharing target and clicked "remember" -> open by default -> click clear.

![image](https://user-images.githubusercontent.com/10332458/142015295-566a113a-0024-4475-842b-3b92287aca0d.png)

**Comment by starlite-jack** [comment]
> In your phone's settings. Go to the settings of the application that you selected as sharing target and clicked "remember" -> open by default -> click clear.
> 
> ![image](https://user-images.githubusercontent.com/10332458/142015295-566a113a-0024-4475-842b-3b92287aca0d.png)

I've already tried it and it shows me this
![Edited_20211117_054037~2](https://user-images.githubusercontent.com/90665508/142078341-3b829f6e-e27b-44f8-b20e-8100d4894bbc.jpg)
what do I do now?

**Comment by K1rakishou** [comment]
Did you try to share stuff into KurobaEx? Otherwise, you need to go to the settings of the application that you selected when clicked shared media content, not KurobaEx's settings.

**Comment by starlite-jack** [comment]
> Did you try to share stuff into KurobaEx?

no, I didn't. I just share a media using kurobaex. at first, it gives me the option to share it different apps. 

what I did is I set a default app when I'm sharing an image because every time kurobaex shares a media, it always ask me to set a default app for sharing the media. so i set a default app to share a media.

my question is, how to undo this option?
